src/less/infotip/base/infotip.less

```diff
@@ -111,12 +111,16 @@ span.infotip__heading {
 .infotip .icon-btn {
     flex-shrink: 0; // todo: Should move to icon-btn in next major
     height: 20px;
-    margin-left: 16px;
     outline-offset: 2px;
     overflow: visible;
     width: 20px;
 }
 
+button.infotip__close,
```

Comment:
Do we need both selectors? Just `.infotip__close` should be enough, right?

Reply:
well not quite. the issue is that the icon-button has this class,
```
button.icon-btn,
a.icon-link {
    background-color: transparent;
    border: 2px solid transparent;
    box-sizing: border-box;
    font-family: inherit;
    height: 40px;
    margin: 0;
    padding: 0;
    vertical-align: text-bottom;
    width: 40px;

    > svg {
        .customFillProperty(icon-button-icon-foreground-color);

        max-width: 100%; // helps when width & height HTML attrs not set
        position: relative; // Safari centering
    }

    &:active > svg {
        .customFillProperty(icon-button-icon-active-foreground-color);
    }

    &:focus > svg,
    &:hover > svg {
        .customFillProperty(icon-button-icon-hover-foreground-color);
    }

    &:not(:focus-visible) {
        outline: none;
    }
}
```

and because it sets `margin: 0` we need a more specific selector. If not, the `margin: 0` here overrides the 
`margin-left: 16px` that we put on the `infotip__close`.

Reply:
Gotcha. Okay let's keep it! Thanks